static/js/components/AutoComplete.js

```diff
@@ -393,7 +405,8 @@ class AutoComplete extends Component {
       triggerUpdateOnFocus, // eslint-disable-line no-unused-vars
       openOnFocus, // eslint-disable-line no-unused-vars
       maxSearchResults,
-      dataSource,
+      dataSource, // eslint-disable-line no-unused-vars
+      menuHeight,
```

Comment:
I'll add it back to `menuStyle` but we still need to pass it as a separate prop since `maxHeight` is not the same thing as `VirtualScroll`'s height, and it seems clunky to extract that information from a CSS style object